gql/transport/requests.py

```diff
@@ -8,21 +8,33 @@
 
 
 class RequestsHTTPTransport(HTTPTransport):
-    def __init__(self, url, auth=None, **kwargs):
+    def __init__(self, url, auth=None, use_json=False, timeout=None, **kwargs):
+        """
+        :param url: The GraphQL URL
+        :param auth: Auth tuple or callable to enable Basic/Digest/Custom HTTP Auth
+        :param use_json: Send request body as JSON instead of form-urlencoded
+        :param timeout: Specifies a default timeout for requests (Default: None)
+        """
         super(RequestsHTTPTransport, self).__init__(url, **kwargs)
         self.auth = auth
+        self.default_timeout = timeout
+        self.use_json = use_json
 
-    def execute(self, document, variable_values=None):
+    def execute(self, document, variable_values=None, timeout=None):
```

Comment:
@syrusakbary I have this use case:
I create a Client with default timeout of 360. It works in most cases but there's this 1 big call I'm making that takes a long time. I want to give that call 10 minutes timeout.

So specifically when executing that call I want to override the default timeout...  creating a separate Client for it is unnecessary...

`Client` already defines execute as `def execute(self, document, *args, **kwargs):` and forwards all the `kwargs` to the transport...  So I don't see why transports should not support custom kwargs